includes/im_user_userheader_block.inc

```diff
@@ -443,9 +442,9 @@ if(!empty ($user_fields->field_user_stores['und'][0]['value'])) {
 			              ->execute();
 			             
 					    $storeRegion = $fetchRegion->fetchfield();		   

```

Comment:
you have a problem with your design if your code is indented this much - consider refactoring `EXTRACT_METHOD`